src/bot.py

```
@@ -6,784 +6,740 @@
+            else:
+                content = formatted_text
+
+            if content:
+                if node.role == "user":
+                    user_payload: ChatCompletionUserMessageParam = {"role": "user", "content": content}
+                    if accept_usernames and node.user_id:
+                        sanitized_name = REGEX_USER_NAME_SANITIZER.sub("", node.user_display_name or "")[:64]
+                        user_payload["name"] = sanitized_name or str(node.user_id)
+                    messages_payload.append(user_payload)
+                elif node.role == "assistant":
+                    assistant_text = formatted_text if isinstance(content, str) else formatted_text
```

Ruff (0.14.10)

Useless `if`-`else` condition

(RUF034)